backend/src/users/user-relationship/entities/user-relationship.entity.ts

```diff
@@ -0,0 +1,26 @@
+import {
+	Column,
+	Entity,
+	JoinColumn,
+	OneToMany,
+	PrimaryGeneratedColumn,
+} from 'typeorm';
+import { User } from '../../user/entities/user.entity';
+
+export enum validRelationships {
+	FRIEND = 'friend',
+	BLOCKED = 'blocked',
+}
```

Comment:
Do you know what exactly is saved in the DB here? Is it the string 'friend', or some int value?

Reply:
also related to update-user-relationship.dto.ts btw, since there it is stored

Reply:
asking because it might be annoying to store the string 'friend' for every user-relationship, since it's then tightly coupled with the exact word "friend"

could also be very well "out of scope" and not relevant..

Reply:
It is a string that i stored atm, I can be changed to another relationship with a relations table containing all valid relationship, and then just creating a relationship on id's? But seems like over-kill, could also be another ticket for crating the relationship entity, endpoints and relationships.  I feel like the enum kinda does the same thing, but indeed more tightly coupled.
Do you have other suggestions? 
And do we want to create another table? :....... (

Reply:
I guess, as long as it's restricted by typeorm that we can only fill in 'friend' or 'blocked' (or anything for the enum) than it's fine. Then we can always display something else if we want to :P